src/Raven.Server/Utils/UnhandledExceptions.cs

```diff
@@ -32,13 +32,13 @@ public static void Track(RavenLogger logger)
 
             TaskScheduler.UnobservedTaskException += (sender, args) =>
             {
-                if (logger.IsInfoEnabled == false)
+                if (logger.IsDebugEnabled == false)
```

Comment:
can we tell websocket ones from others? If we put everything in Debug, we might miss an exception leak somewhere

Reply:
We discussed with @gregolsky and agreed that this change returns back 6.2 behavior and we will keep the issue open to look deeper into why we are getting WebSocketException at all (was not reproducing on my setup).